scripts/roboclawtest.py

```diff
@@ -24,7 +24,7 @@
         print(roboclaw0.ReadVersion(address))
         print(roboclaw0.ReadEncM1(address))
     elif connected1:
-        print("Connected to /dev/serial1.")
+        print("Connected to /dev/serial1. Only proceed if you know what you're doing, otherwise ask for help on Slack.")
```

Comment:
whats this slack?

Reply:
https://github.com/nasa-jpl/open-source-rover/?tab=readme-ov-file#join-a-community-of-hundreds-of-builders